src/subscribr.js

```diff
@@ -55,6 +55,12 @@ module.exports = class Subscribr {
 
 }
 
+let idCounter = 0;
```

Comment:
Underscore I think have something like that

http://underscorejs.org/#uniqueId

But I think this could break the quantum node